ament_tools/verbs/test_results/cli.py

```diff
@@ -160,12 +162,12 @@ def print_summary(results, show_stable=False, show_unstable=True):
     :param show_stable: print tests without errors or failures
     :param show_unstable: print tests with errors or failures
     """
-    def callback(name, num_tests, num_errors, num_failures):
+    def callback(name, num_tests, num_errors, num_failures, num_skipped):
         if show_stable and not num_errors and not num_failures:
-            print('%s: %d tests' % (name, num_tests))
-        if show_unstable and (num_errors or num_failures):
-            print('%s: %d tests, %d errors, %d failures' %
-                  (name, num_tests, num_errors, num_failures))
-    sum_tests, sum_errors, sum_failures = aggregate_results(results, callback)
-    print('Summary: %d tests, %d errors, %d failures' %
-          (sum_tests, sum_errors, sum_failures))
+            print('%s: %d tests, %d skipped' % (name, num_tests, num_skipped))
+        if show_unstable and (num_errors or num_failures or num_skipped):
+            print('%s: %d tests, %d skipped, %d errors, %d failures' %
+                  (name, num_tests, num_skipped, num_errors, num_failures))
```

Comment:
With the current logic it seems that both pint statements (line 167 as well as 169) could happen since the second part of the boolean condition is not the exact inverse anymore?